src/css-loader-core/loader.js

```diff
@@ -1,112 +1,32 @@
-// Copied from https://github.com/css-modules/css-modules-loader-core
-
-import postcss from "postcss";
 import fs from "fs";
 import path from "path";
 
-import Parser from "./parser";
-
-class Core {
-  constructor(plugins) {
-    this.plugins = plugins || Core.defaultPlugins;
+export function resolveRelativeImport(importee, importer, projectRoot) {
```

Comment:
It seems to be an internal function. Let's not export it then

Reply:
ah, of course. good catch :)